### ISSUE TITLE
Cannot decrypt test wallet with known password

### ISSUE THREAD

**Issue by TTN-** [gurnec/btcrecover#65]
To confirm the setup I am running actually works, I decided to create a test wallet with the password `Testing123`

I cd to my .armory folder (Ubuntu 16.04)
`python btcrecover.py --wallet testing123_armory_29rBFk1q9_encrypt.wallet` 

I type my password, and it comes back as exhausted. I can go into armory, and enter the password and it works. The wallet was made on armory v0.93.x (pretty sure it was .1) wallet version is 1.35

Attached is the wallet in case someone would like to test, I had to zip it to get it attached.
[testing123_armory_29rBFk1q9_encrypt.wallet.zip](https://github.com/gurnec/btcrecover/files/1053880/testing123_armory_29rBFk1q9_encrypt.wallet.zip)

**Comment by TTN-** [comment]
Am I doing something wrong? I found this to be quite shocking. I've also attempted to find the password using tokens for it but it won't work either.

**Comment by gurnec** [comment]
I can't reproduce being able to open your attached wallet file with Armory, I've tried:

 * 0.93.3 64bit, Ubuntu 16.04
 * 0.94.1 64bit, Windows 10
 * 0.96 64bit, Ubuntu 16.04

...and all three complain of "Incorrect passphrase for wallet".

Could you please download [the attached script](https://github.com/gurnec/btcrecover/files/1056373/ttn.py.txt), and run it on the machine containing Armory and your Armory wallet file (only runs on Linux)? You'll probably need to change the filename on line 12 to get it to run correctly. You don't need btcrecover on the machine, just Armory.

Thanks!

(and in case it's not obvious, don't run this on any "real" wallets--it will reveal the private keys)

**Comment by TTN-** [comment]
I see. I have deleted the wallet, imported it and now the password will not work. I simply created another wallet with the same password, deleted it, imported it, and it will unlock. Next I ran the btcrecovery program in it and it was able to find that testing123 was the correct password!

Sorry it appears my test wallet was broken somehow (?) but clearly your software works. Sorry to waste your time and thank you for responding!
